libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/app/SalesforceHybridSDKManager.java

```diff
@@ -29,6 +29,8 @@
 import android.app.Activity;
 import android.content.Context;
 
+import androidx.annotation.NonNull;
```

Comment:
Everything in this source is inspector cleanup for existing trivial issues and those introduced by the Kotlin conversion.